composer/plugins/gcs_plugin/operators/gcs_to_gcs.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2018 Google LLC
```

Comment:
Please remove the operator as well.

If we cannot, please retain the original license headers (we should definitely not replace any license headers with Google copyright) and move to a third_party/ directory per go/thirdparty#projects